### ISSUE TITLE
[FEATURE] Chat Agent using PGVector

### ISSUE THREAD

**Issue by shyamramath** [langchain4j/langchain4j#2347]
**Is your feature request related to a problem? Please describe.**
I wanted to create a chat agent using PGVector store . I wanted to combine the project customer-support-agent-example and the Langchain4j PGvector example project . Is that something that can be done ? 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by langchain4j** [comment]
@shyamramath yes, you can change `InMemoryEmbeddingStore` to `PgVectorEmbeddingStore` in the example [here](https://github.com/langchain4j/langchain4j-examples/blob/de760ae8020d498554abd99cac65c607bf3c1ea3/customer-support-agent-example/src/main/java/dev/langchain4j/example/CustomerSupportAgentApplication.java#L100).

[Here](https://github.com/langchain4j/langchain4j-examples/blob/main/pgvector-example/src/main/java/PgVectorEmbeddingStoreExample.java) is an example of how to create `PgVectorEmbeddingStore`.
